tutor/src/sandbox.jsx

```diff
@@ -0,0 +1,25 @@
+const React = require('react');
+const BS = require('react-bootstrap');
+
+const Sandbox = React.createClass({
+  render: () => {
+    return (
```

Comment:
If/when we move to es6, we should discuss fat arrow usage.  For instance here, it seems like it'd be unwanted here since it binds to `this`, which `React.createClass` is already doing for us.  Neither binding is needed for `render`, it's not called from a callback.  I bet we're essentially nesting 3 functions deep here.